docs/content/administration/reverse-proxies.en-us.md

```diff
@@ -381,3 +381,9 @@ If you really need to do so, to make Gitea works with sub-path (eg: `http://exam
 1. Set `[server] ROOT_URL = http://example.com/gitea/` in your `app.ini` file.
 2. Make the reverse-proxy pass `http://example.com/gitea/foo` to `http://gitea-server:3000/foo`.
 3. Make sure the reverse-proxy not decode the URI, the request `http://example.com/gitea/a%2Fb` should be passed as `http://gitea-server:3000/a%2Fb`.
+
+## Docker / Container Registry
```

Comment:
```suggestion
## Docker container Registry
```

Reply:
I think the original text means docker or other container registry like podman and etc. since the protocol is compatible.

Reply:
`Docker / Container Registry` is the right wording.